src/ui/components/SettingsPage.tsx

```diff
@@ -62,6 +63,22 @@ const SettingsPage: FC = () => {
     [collectionId, customCollection, setCustomCollection],
   );
 
+  const handleUpdateOptionChange = useCallback(
+    (event: ChangeEvent<HTMLInputElement>) => {
+      setUpdateOption(event.target.value);
+    },
+    [],
+  );
+
+  const handleUpdateClick = useCallback(async () => {
+    try {
+      const options = updateOption === "sync" ? true : false;
+      await browser.experiments.nimbus.updateRecipes(options);
```

Comment:
here you're calling it options but the arg is called forceSync. Lets call this forceSync too.